pkg/ocm/config.go

```diff
@@ -105,6 +105,12 @@ func GetEnv() (string, error) {
 				}
 			}
 		}
+	} else { // Go back to exact URL check (in case of other URLs like local envs, etc.)
```

Comment:
I'd say we can move [this](https://github.com/openshift/rosa/blob/4d39081f7312cceb0e90e8ac3503d5ab12aa862c/pkg/ocm/config.go#L116) comment up [here](https://github.com/openshift/rosa/blob/4d39081f7312cceb0e90e8ac3503d5ab12aa862c/pkg/ocm/config.go#L93) and just remove the else if [this](https://github.com/openshift/rosa/blob/4d39081f7312cceb0e90e8ac3503d5ab12aa862c/pkg/ocm/config.go#L117) uses `range urlAliases` instead

Reply:
@gdbranco Moved the FedRAMP logic upwards and removed else condition